operator-kustomize/deployment.yaml

```diff
@@ -18,6 +18,9 @@ spec:
         - name: minio-operator
           image: minio/k8s-operator:v3.0.5
           imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              memory: 64Mi
```

Comment:
Seems a bit stingy? The operator also maintains a cache locally of k8s API responses. Maybe 500megs or 1gig? Not sure how the k8s client sizes its cache.

Reply:
We have a cache? I think ging to 256Mb like @harshavardhana suggest is a good balance

Reply:
The k8s client does have caching internally, so far we have been testing with tiny k8s clusters and I am not sure how the memory foot print changes on larger clusters.

Reply:
We don't see Operator consume large amounts of memory, even at 256 seems too much memory to spare from the usual 11MB we see in development